comms/rabbitmq.go

```diff
@@ -64,10 +64,10 @@ func connectRabbitMQ(queueUrl string) (*amqp.Connection, error) {
 func (rmq rabbitMQComms) AdvertiseAgent(me defs.AgentAdvert) error {
 
 	queueUrl := fmt.Sprintf("amqp://%s:%s@%s:%s/",
-		rmq.config.AMQP.User,
-		rmq.config.AMQP.Password,
-		rmq.config.AMQP.Host,
-		rmq.config.AMQP.Port,
+		rmq.config.Comms.User,
+		rmq.config.Comms.Password,
+		rmq.config.Comms.Host,
+		rmq.config.Comms.Port,
 	)
```

Comment:
You could move this to be a method on `rabbitMQComms` or make `connectRabbitMQ()` a method and move it there.